### DISCUSSION TITLE
srsRAN Project 23.10

### DISCUSSION THREAD

**Discussion by andrepuschmann** [srsran/srsRAN_Project#266] [Announcements]
Hey everyone,
as some of you might have already noticed we've just released srsRAN Project 23.10 - another milestone that we are very happy to share with the open-source RAN community.

Most of the new features have already been made available publicly and we've shared detailed information with you through various posts, including the addition of [MIMO](https://github.com/srsran/srsRAN_Project/discussions/102) as well as the [Open Front Haul (OFH)](https://github.com/srsran/srsRAN_Project/discussions/90) library. The addition of this collection of features marks a milestone in our development roadmap and so we've decided to make this a tagged release.

Definitely worth mentioning here is that this release now has support for up to four MIMO layers in the downlink. This let's the downlink rate for a single user peak at approx 1.4Gbit/s.
Understandably, those rates are only achievable if the entire system is properly configured and we do plan to share a detailed performance assessment and tuning guide with you in the near future.

Another very exciting addition we've first introduced this summer is the [E2 interface](https://github.com/srsran/srsRAN_Project/discussions/163). It now has support for the Key Performance Measurement (KPM) and the RAN Control (RC) service model. Development here is still very much ongoing so do expect more things to come over the coming weeks.

Something extremely useful that we've actually not blogged about before is the addition of [Docker](https://github.com/srsran/srsRAN_Project/tree/main/docker) containers along with scripts that let you setup an end-to-end system with RAN and 5GC with literally a single command. Have a look and give it a try yourself.

Please check out the high-level changelog [here](https://github.com/srsran/srsRAN_Project/releases/tag/release_23_10) - or for detail fanatics the 2745 commits counting git history.

We hope you enjoy the release - Happy hacking everyone!

P.S. And maybe even see some of you guys at the [srsRAN Project Fall workshop](https://github.com/srsran/srsRAN_Project/discussions/145) starting tomorrow October 23th in Arlington, VA.

**Comment by rowney24** [comment]
Thank you very much. Definitely going to be an exciting week in the lab. I hope there will be config files for the n321 or any SDRs that support these very high data rates so that we can get started with it.

**Comment by improperaffo** [comment]
Any news about the workshop's recordings? Really looking forward to them
